content/github/managing-security-vulnerabilities/troubleshooting-dependabot-errors.md

```diff
@@ -88,3 +88,5 @@ If you unblock {% data variables.product.prodname_dependabot %}, you can manuall
 
 - **Security updates**—display the {% data variables.product.prodname_dependabot %} alert that shows the error you have fixed and click **Create {% data variables.product.prodname_dependabot %} security update**.
 - **Version updates**—display the log file for the manifest that shows the error that you have fixed and click **Check for updates**.
+
+You can also visit the **Insights** / **Dependency Graph** / **Dependabot** tab for the repository, and click "Check for updates".
```

Comment:
@dfrankow - Many thanks for raising this PR (and the associated issue). Your proposed change highlights that the original text wasn't clear. Your suggestion is a clarification of the original instruction rather than an alternative way of triggering a manual version check. 

My suggestion below changes replaces the original text and alters your suggested text to keep it in line with the description on this page: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/listing-dependencies-configured-for-version-updates#viewing-dependabot-log-files

Please click to commit my suggestion if you agree and we can get this merged. Thanks again for contributing - it'll be good to clarify this point.  🎉 

```suggestion
- **Version updates**—on the **Insights** tab for the repository click **Dependency graph**, and then click the **Dependabot** tab. Click **Last checked *TIME* ago** to see the log file that {% data variables.product.prodname_dependabot %} generated during the last check for version updates. Click **Check for updates**.
```

Reply:
When I clicked on your suggestion, it shows screenshots, which is amazing and much better than my update. Thank you!